cirq/testing/_compat_test_data/module_a/__init__.py

```diff
@@ -0,0 +1,11 @@
+from logging import info
+
+from cirq.testing._compat_test_data.module_a import (
+    module_b,
+)
+
+from cirq.testing._compat_test_data.module_a.dupe import (
+    DUPE_CONSTANT,
+)
+
```

Comment:
I would add a line like `test_value = 'module_a'` to all these files to use in the test assertions.

Reply:
👍 Added.